AjobK/Athena
# [277] Post reading time

closes #277 

<br />

 **What is it supposed to do**  
Add a utility for calculating reading times. Should also display reading times for posts on the home screen.

<br />

 **How can we test your branch**  
Have a look at the home page, add a couple of posts of different sizes and check if the time is correct. WORDS_PER_MIN is set to 265 as suggested by Medium.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select atleast two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
